scripts/rebuild-dna.sh

```diff
@@ -3,6 +3,6 @@
 npm run dna-pack
 cp dna/workdir/profiles.dna electron/binaries/profiles.dna
 cp dna/workdir/projects.dna electron/binaries/projects.dna
-rm -rf tmp2
-rm -rf tmp
+rm -rf user-data
+rm -rf user2-data
```

Comment:
this file also could be deleted because its purpose is replaced by the new dna-reset command